gcc/d/patches/patch-toplev-ddmd-9.patch

```diff
@@ -1692,18 +1740,158 @@ This implements building of self hosted D compiler in GCC.
  
  
  .PHONY: check-target-libphobos maybe-check-target-libphobos
-@@ -52282,7 +53858,6 @@ configure-target-libatomic: stage_curren
- @if target-libatomic
- maybe-configure-target-libatomic: configure-target-libatomic
- configure-target-libatomic: 
--	@: $(MAKE); $(unstage)
- 	@r=`${PWD_COMMAND}`; export r; \
- 	s=`cd $(srcdir); ${PWD_COMMAND}`; export s; \
- 	echo "Checking multilib configuration for libatomic..."; \
-@@ -52320,6 +53895,412 @@ configure-target-libatomic:
- 
- 
+@@ -52175,118 +53751,520 @@ maybe-mostlyclean-target-libitm:
+ @if target-libitm
+ maybe-mostlyclean-target-libitm: mostlyclean-target-libitm
  
+-mostlyclean-target-libitm: 
+-	@: $(MAKE); $(unstage)
+-	@[ -f $(TARGET_SUBDIR)/libitm/Makefile ] || exit 0; \
+-	r=`${PWD_COMMAND}`; export r; \
++mostlyclean-target-libitm: 
```

Comment:
I wonder where this libitm stuff came from.

Reply:
This snippet is from the original Makefile.in. My guess is that patch tries to reorder some newly added blocks in the Makefile. I should probably apply some parts of this patch manually and regenerate.

Reply:
:( Looks like a bug in diffutils 3.3. diff 3.6 produces normal output.